client.py

```diff
@@ -4,19 +4,34 @@
 from hazelcast.discovery import HazelcastCloudDiscovery
 
 """
-This is boilerplate application that configures client to connect Hazelcast Cloud cluster.
-After successful connection, it puts random entries into the map.
+This is a boilerplate client application that connects to your Hazelcast Viridian cluster.
+See: https://docs.hazelcast.com/cloud/get-started
 
-See: https://docs.hazelcast.cloud/docs/python-client
+Snippets of this code are included as examples in our documentation,
+using the tag:: comments.
 """
 
 logging.basicConfig(level=logging.INFO)
+# tag::env[]
```

Comment:
So, will the user see this comment when they download the code samples from the cloud?

Reply:
I added this so we can filter out that line. But if our release process already does that, then I think we can remove it.